rpcapi.yaml

```diff
@@ -457,6 +487,43 @@ components:
                             stream:
                                 type: string
 
+        PaginatedRpcPersonList:
+            type: object
+            required:
+                - count
+                - results
+            properties:
+                count:
+                    type: integer
+                    example: 123
+                next:
+                    type: string
+                    nullable: true
+                    format: uri
+                    example: https://api.example.org/accounts/?offset=400&limit=100
+                previous:
+                    type: string
+                    nullable: true
+                    format: uri
+                    example: https://api.example.org/accounts/?offset=200&limit=100
+                results:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/RpcPerson'
+
+        RpcPerson:
```

Comment:
You can avoid defining a new schema by using `Person` model.

Reply:
Did it this way to reduce mixing generated and hand-written schema parts, but the schemas are compatible so you're right, we should be optimistic. :-) This also avoids leaking the `RpcPerson` serializer name into the API.